Makefile

```diff
@@ -1,5 +1,5 @@
 build:
-	GOFLAGS="-mod=vendor" go build -ldflags "-X github.com/openshift/release-controller/vendor/k8s.io/client-go/pkg/version.gitCommit=$$(git rev-parse HEAD) -X github.com/openshift/release-controller/vendor/k8s.io/client-go/pkg/version.gitVersion=v1.0.0+$$(git rev-parse --short=7 HEAD)" ./cmd/release-controller
+	hack/build.sh
```

Comment:
This change doesn't belong in here.

Reply:
Why not?

Reply:
Adding the new ldflags should happen at the same time that we start using the upstream version package.